Static/Row.swift

```diff
@@ -134,7 +140,12 @@ public struct Row: Hashable, Equatable {
     // MARK: - Initializers
 
     public init(text: String? = nil, detailText: String? = nil, selection: Selection? = nil,
-        image: UIImage? = nil, accessory: Accessory = .None, cellClass: CellType.Type? = nil, context: Context? = nil, editActions: [EditAction] = [], UUID: String = NSUUID().UUIDString) {
+        image: UIImage? = nil, accessory: Accessory = .None, cellClass: CellType.Type? = nil, cellNib: UINib? = nil, height: CGFloat? = nil, context: Context? = nil, editActions: [EditAction] = [], UUID: String = NSUUID().UUIDString) {
+        
+        if let _ = cellNib where cellClass != nil {
+            assert(false, "Specifying a cell Nib requires specifying a custom cell Class too")
```

Comment:
Should the error for the assert be `You cannot specify both a nib and a custom cell class`?